lib/ex_sdp/attribute/fmtp.ex

```diff
@@ -163,6 +163,22 @@ defmodule ExSDP.Attribute.FMTP do
          do: {rest, %{fmtp | profile_id: value}}
   end
 
+  defp parse_param(["apt=" <> value | rest], fmtp), do: {rest, Map.put(fmtp, :apt, value)}
+
+  defp parse_param(["repair-window=" <> value | rest], fmtp),
+    do: {rest, Map.put(fmtp, :repair_window, value)}
+
+  defp parse_param([head | rest], fmtp) do
+    [start_range, end_range] = String.split(head, "-")
+
+    with {:ok, start_range} <- Utils.parse_numeric_string(start_range),
+         {:ok, end_range} <- Utils.parse_numeric_string(end_range) do
+      {rest, Map.put(fmtp, :redundancy_range, "#{start_range}-#{end_range}")}
```

Comment:
let's store that in a usable way, for example a `{start_range, end_range}` tuple